components/todo/TodoContainer.vue

```diff
@@ -0,0 +1,75 @@
+<template>
+  <div class="todo-container">
+    <h2>Add Task</h2>
+    <div class="todo__header">
+      <button @click.prevent="$store.commit('todos/setCreate')">Create</button>
```

Comment:
while we do use `commit` directly outside of store, it a good practice to only trigger `actions` from components.

Reply:
Noted.